MGMT-13955: Add known issue regarding ignoring disk size validations (#5158)
openshift/assisted-service: docs/enhancements/api-for-ignoring-validations.md

* We will have to be careful when examining bug reports and triage tickets to ensure we are aware of when this feature has been used.
* Might allow some customers to ignore things they shouldn't ignore.

### Known issues
* If the user provides disks of less than the required size then even though the validation has been bypassed, this will not work

>   - This is because the primary disk for the node could not be chosen, minimum disk size is used as part of the definition of `eligible` disks.

Reviewer: ```suggestion
  - This is because the installation disk for the node could cannot be chosen, minimum disk size is used as part of the definition of `eligible` disks and the installation disk has to be eligible. Since eligibility checks are not  a "validation", skipping validations will not not alleviate this issue
```

>   - This is because the installation disk for the node cannot be chosen, minimum disk size is used as part of the definition of `eligible` disks and the installation disk has to be eligible. Since eligibility checks are not a "validation", skipping validations will not alleviate this issue

Reviewer: ```suggestion
  - This is because the installation disk for the node cannot be chosen, minimum disk size is used as part of the definition of "eligible" disks and the installation disk has to be eligible. Since eligibility checks are not a "validation", skipping validations will not alleviate this issue
```

>   - This is because the installation disk for the node could cannot be chosen, minimum disk size is used as part of the definition of `eligible` disks and the installation disk has to be eligible. Since eligibility checks are not a "validation", skipping validations will not alleviate this issue

Reviewer: ```suggestion
  - This is because the installation disk for the node cannot be chosen, minimum disk size is used as part of the definition of `eligible` disks and the installation disk has to be eligible. Since eligibility checks are not a "validation", skipping validations will not alleviate this issue
```

>   - This is because the installation disk for the node could cannot be chosen, minimum disk size is used as part of the definition of `eligible` disks and the installation disk has to be eligible. Since eligibility checks are not  a "validation", skipping validations will not not alleviate this issue

Reviewer: ```suggestion
  - This is because the installation disk for the node could cannot be chosen, minimum disk size is used as part of the definition of `eligible` disks and the installation disk has to be eligible. Since eligibility checks are not a "validation", skipping validations will not alleviate this issue
```
